src/index.html

```diff
@@ -17,11 +21,360 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
     />
   </head>
+  <body class="page">
+    <header class="header">
+      <a href="#HOME">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link nav__link--is-active"
```

Comment:
Not fixed, redundant class
```suggestion
              class="is-active nav__link"
```
<img width="833" alt="image" src="https://github.com/user-attachments/assets/38e2e1d1-5039-453a-a2e2-560d1b7cf91d" />